views/partials/subscribe_modal/subscribe_to_ffxm.hbs

```diff
@@ -12,9 +12,9 @@
       <label for="additional-emails-checkbox">Get the lastest news from Mozilla and Firefox.</label>
     </div>
     <div class="input-group-button">
-      <img class="loader" src="/img/loader.gif" alt="loading data" />
-      <input id="subscribe-submit" type="submit" class="button submit-email transparent-button" value="Submit" tabindex="0" />
+      <img class="loader" src="img/loader.gif" alt="loading data" />
```

Comment:
Nit: this went from "/img/" to "img/" (no leading "/"), which could cause problems depending on where this partial gets used.

All other instances of the loader.gif seem to still use the absolute `"/img/loader.gif"` src.